src/CompositeNormL1.jl

```diff
@@ -0,0 +1,39 @@
+# Composition of the L1 norm with a function
+export CompositeNormL1
+
+@doc raw"""
+    CompositeNormL1(h,c!,J!,A,b)
+
+Returns the ``\ell_{1}`` norm operator composed with a function
+```math
+f(x) = λ ||c(x)||_1
+```
+where ``\lambda > 0``.
+"""
+mutable struct CompositeNormL1{
+    R <: Real,
+    V0 <: Function,
+    V1 <: Function,
+    V2 <: AbstractMatrix{R},
+    V3 <: AbstractVector{R},
+  } <: CompositeProximableFunction
+    h::NormL1{R}
+    c!::V0
+    J!::V1
+    A::V2
+    b::V3
+    function CompositeNormL1(
+      h::NormL1{R},
+      c!::Function,
+      J!::Function,
+      A::AbstractMatrix{R},
+      b::AbstractVector{R},
+    ) where {R <: Real}
+      length(b) == size(A,1) || error("Composite Norm L1 : Wrong input dimensions, constraints should have same length as rows of the jacobian")  
+      new{R,typeof(c!),typeof(J!),typeof(A),typeof(b)}(h,c!,J!,A,b)
```

Comment:
```suggestion
      new{R, typeof(c!), typeof(J!), typeof(A), typeof(b)}(h, c!, J!, A, b)
```
Use spacing everywhere for readability.